src/openrct2/scenario/Scenario.h

```diff
@@ -363,6 +363,9 @@ enum
 #define AUTOSAVE_PAUSE 0
 #define DEFAULT_NUM_AUTOSAVES_TO_KEEP 10
 
+// Defines how many tiles from the edge duck target coordinates should be generated
+#define DUCK_TARGET_TILES_AWAY_FROM_EDGE 3
```

Comment:
What is the reason you divided the original number by two?

Reply:
Currently the range you select goes like following:
```
start = 1 + DUCK_TARGET_TILES_AWAY_FROM_EDGE
end = MAXIMUM_MAP_SIZE_PRACTICAL - (DUCK_TARGET_TILES_AWAY_FROM_EDGE * 2)
```
So why do you remove twice as much from the end but not from the beginning?

Reply:
Because I add an offset of 1 + DUCK_TARGET_TILES_AWAY_FROM_EDGE at the beginning to make the generated coordinates start at the correct number. If I didn't remove DUCK_TARGET_TILES_AWAY_FROM_EDGE twice at the end, the generated coordinates would have a maximum of 254.